### PR TITLE
Fix snapshot test

### PR DESCRIPTION
**Pull request by jingxu97**
Remove snapshot invalid token test, modify the workflow for creating a
snapshot with the same name but different volume source id in order to
use valid volume id.

### CODE REVIEW

**Comment by jingxu97** [comment]
cc @wackxu @xing-yang

File: pkg/sanity/controller.go [hunk 1]
```diff
@@ -1254,21 +1254,6 @@ var _ = DescribeSanity("ListSnapshots [Controller Server]", func(sc *SanityConte
...
-			},
-		)
-		Expect(err).To(HaveOccurred())
-		Expect(vols).To(BeNil())
-
-		serverError, ok := status.FromError(err)
-		Expect(ok).To(BeTrue())
-		Expect(serverError.Code()).To(Equal(codes.Aborted))
-	})
-
```
Comment (xing-yang) [inline pkg/sanity/controller.go]: I think we still need some kind of invalid token test?  I thought you want to remove the next test "should fail when the starting_token is greater than total number of snapshots".

Comment (xing-yang) [reply pkg/sanity/controller.go]: Just talked to Jing.  We don't know what kind of token a driver will use so we don't really know what is invalid.  So this test can be removed.

**Comment by jingxu97** [comment]
cc @lpabon @saad-ali Could you please help approve? Thanks!

**Pull request merged**